shell/platform/darwin/ios/framework/Source/FlutterEngineTest.mm

```diff
@@ -28,7 +28,7 @@ - (void)tearDown {
 }
 
 - (void)testCreate {
-  id project = OCMClassMock([FlutterDartProject class]);
+  FlutterDartProject* project = [[FlutterDartProject alloc] init];
```

Comment:
@jmagman The test failure was due to the mock project doesn't have the setting ivar mocked. For these tests, I either removed the project completely or used a real project object. There was no reason to mock the project at the first place.